src/view/components/rawDataElementSelector.styl

```diff
@@ -11,5 +11,5 @@ governing permissions and limitations under the License.
 */
 
 .DataElementSelector-buttonAdjustedForLabel {
-  margin-top: 24px;
+  margin-top: 24px !important;
```

Comment:
I don't understand why this is necessary.

Reply:
I dug into this. It turns out the styles from `@adobe/react-spectrum` are bundled in our CSS file before `.DataElementSelector-buttonAdjustedForLabel`, which is good, but the styles from `@react-spectrum/combobox` are bundled into the CSS file _after_ `.DataElementSelector-buttonAdjustedForLabel`. The combobox styles include a `._spectrum-ActionButton_7a745,._spectrum-Button_7a745,._spectrum-ClearButton_7a745,._spectrum-FieldButton_7a745,._spectrum-LogicButton_7a745` selector that sets margin to 0. Since this selector comes after `.DataElementSelector-buttonAdjustedForLabel`, its 0 margin wins.

I don't think there's a great way of forcing the ComboBox's styles to be bundled into the CSS file before our own styles (I've asked on the react-spectrum channel. I'll let you know if I hear otherwise.). Considering this, using `!important` here is a reasonable workaround, but I think it would be better to add a `marginTop="size-300"` prop to the button inside `rawDataElementSelector.jsx` instead of having this class. I probably should have done that in the first place.